src/components/CreateNewTaskItem.js

```diff
@@ -0,0 +1,32 @@
+import Task from '../modules/Task.js';
+import DataStore from '../modules/DataStore.js';
+
+import renderTaskItem, { refreshTaskList } from './TaskListItem.js';
+
+const formAddTask = () => {
+  const form = document.createElement('form');
+  form.id = 'form-add-task';
+  form.innerHTML = `<input type="text" name="newTask" placeholder="Add new task ..." required/>
```

Comment:
- [ ] **Optional:** Consider clearing the input field every time an item is added to the list, this would be better for UX.